servlet/src/main/scala/org/http4s/servlet/syntax/ServletContextSyntax.scala

```diff
@@ -38,23 +39,43 @@ final class ServletContextOps private[syntax] (val self: ServletContext) extends
     *
     * Assumes non-blocking servlet IO is available, and thus requires at least Servlet 3.1.
     */
-  def mountService[F[_]: Async](
+  @deprecated("Use mountRoutes instead", "1.0.0-M32")
```

Comment:
Current PR is about to merge to `series/0.23`, so the version should be `0.23.11`
```suggestion
  @deprecated("Use mountRoutes instead", "0.23.11")
```